### DISCUSSION TITLE
Please test Comfast 3000Mbps PCIe

### DISCUSSION THREAD

**Discussion by gitclint** [geerlingguy/raspberry-pi-pcie-devices#361] [General] [answered]
Would it be possible to test a Comfast 3000Mbps Dual band Wifi 6 Intel AX200 PCI-E 1X Network Card?
https://comfastwifi.us/comfast-cf-ax200-3000mbps-dual-band-wifi-6-intel-network-card
I would be happy to pay for the card! Thanks!

**Comment by geerlingguy** [comment, marked as answer]
If it uses the AX200, it _should_ work after installing the Intel firmware and appropriate igw drivers.
